### PR TITLE
Create monitoring vs observability blog

### PR DESCRIPTION
**Pull request by shyamvalsan**
monitoring vs observability blog

### CODE REVIEW

File: blog/2023-10-24-monitoring-vs-observability/index.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+A critical aspect that is often overlooked is the need for real-time, high-fidelity metrics, which are metrics sampled at very high frequencies, often per second. In a system where millions of transactions are happening every minute, a five-minute average could hide critical spikes that may indicate system failure or degradation. Observability platforms are generally better suited to provide this level of granularity than traditional monitoring tools.
+
+##### Logs: Event-Driven in Monitoring vs Queryable in Observability
+
+Logs provide a detailed account of events and are fundamental to both monitoring and observability. However, the treatment differs.
+
+Monitoring: Event-Driven Logs
```
Comment (hugovalente-pm) [inline blog/2023-10-24-monitoring-vs-observability/index.md:62]:
```suggestion
Monitoring: Event-Driven Logs

```

this seems to miss a paragraph on the end-result blog

File: blog/2023-10-24-monitoring-vs-observability/index.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+In monitoring systems, logs are often used for event-driven alerting. For instance, a log entry indicating a failed login attempt might trigger an alert for potential security concerns. These logs are essential but are typically consulted only when an issue has already been flagged by the monitoring system.
```
Comment (hugovalente-pm) [inline blog/2023-10-24-monitoring-vs-observability/index.md:63]:
```suggestion
In monitoring systems, logs are often used for event-driven alerting. For instance, a log entry indicating an elevated permissions login action might trigger an alert for potential security concerns. These logs are essential but are typically consulted only when an issue has already been flagged by the monitoring system.
```

would this example be a better one for a security concern? (idea of a root login)

File: blog/2023-10-24-monitoring-vs-observability/index.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+Observability: Queryable Logs
```
Comment (hugovalente-pm) [inline blog/2023-10-24-monitoring-vs-observability/index.md:65]:
```suggestion
Observability: Queryable Logs

```

this seems to miss a paragraph on the end-result blog

File: blog/2023-10-24-monitoring-vs-observability/index.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+Observability, by contrast, is more proactive. With an observability platform, you’re not just waiting for things to break. You’re continually exploring your data to understand how your system behaves under different conditions. This allows for more preventive measures and enables engineers to understand the system’s behavior deeply.
+
+#### Opinionated Dashboards and Charts
+
+Navigating the sprawling landscape of system data can be a daunting task, particularly as systems scale and evolve. Both monitoring and observability tools offer dashboards and charts as a solution to this challenge, but the philosophy and functionality behind them can differ significantly.
+
+##### Monitoring: Pre-Built and Prescriptive Dashboards
+In the realm of monitoring, dashboards are often pre-built and prescriptive, designed to highlight key performance indicators (KPIs) and metrics that are generally considered important for the majority of use-cases. For instance, a pre-configured dashboard for a database might focus on query performance, CPU usage, and memory consumption. These dashboards serve as a quick way to gauge the health of specific components within your system.
+
+Advantages:
```
Comment (hugovalente-pm) [inline blog/2023-10-24-monitoring-vs-observability/index.md]: this entire bullet/points section needs to be reviewed, I'm not even sure how you wanted to structure it @shyamvalsan  😅

File: blog/2023-10-24-monitoring-vs-observability/index.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+Quick Setup: Pre-built dashboards require little to no configuration, making them quick to deploy.
+Best Practices: These dashboards are often designed based on industry best practices, providing a tried-and-true set of metrics that most organizations should monitor.
+Limitations:
+Lack of Flexibility: Pre-built dashboards are not always tailored to your specific needs and might lack the ability to perform ad-hoc queries or deep dives.
+Surface-Level Insights: While useful for a quick status check, these dashboards may not provide the contextual data needed to understand the root cause of an issue.
+
+##### Observability: Customizable and Exploratory Dashboards
+Contrastingly, observability platforms often allow for much greater customization and flexibility in dashboard creation. You can build your own dashboards that focus on the metrics most relevant to your specific application or business needs. Moreover, you can create ad-hoc queries to explore your data in real-time.
+
+Advantages:
```
Comment (hugovalente-pm) [inline blog/2023-10-24-monitoring-vs-observability/index.md]: same comment as above

**Review by Aliki92 — APPROVED** [review]

File: blog/2023-10-24-monitoring-vs-observability/index.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+
+As systems increasingly shift towards distributed architectures to deliver application services, the roles of monitoring and observability have never been more crucial. Monitoring delivers the situational awareness you need to detect issues, while observability goes a step further, offering the analytical depth to understand the root cause of those issues.
+
+Understanding the nuanced differences between monitoring and observability is crucial for anyone responsible for system health and performance. In dissecting these methodologies, we'll explore their unique strengths, dive into practical applications, and illuminate how to strategically employ each to enhance operational outcomes.
+
+To set the stage, consider a real-world scenario that many of us have encountered: It's 3 a.m., and you get an alert that a critical service is down. Traditional monitoring tools may tell you what's wrong, but they won't necessarily tell you why it's happening leaving that part up to you. With Observability, the tool enables you to explore your system's internal state and uncover the root cause in a faster and easier manner.
```
Comment (hugovalente-pm) [inline blog/2023-10-24-monitoring-vs-observability/index.md:18]:
```suggestion
To set the stage, consider a real-world scenario that many of us have encountered: It's 3 a.m., and you get an alert that a critical service is down. Traditional monitoring tools may tell you what's wrong, but they won't necessarily tell you why it's happening leaving that part up to you. With observability, the tool enables you to explore your system's internal state and uncover the root cause in a faster and easier manner. 
```

throughout the document you use only caps on titles, so suggested change for consistency

File: blog/2023-10-24-monitoring-vs-observability/index.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+Both monitoring and observability rely on data, but the kinds of data they use and how they use it can differ substantially.
+
+##### Metrics in Monitoring vs Observability
+
+Metrics serve as the backbone of both monitoring and observability, providing numerical data that is collected over time. However, the granularity, flexibility, and usage of these metrics differ substantially between the two paradigms.
+
+Monitoring: Predefined and Aggregate Metrics
+In a monitoring setup, metrics are often predefined and tend to be aggregate values, such as averages or sums calculated over a specific time window. These metrics are designed to trigger alerts based on known thresholds. For example, you might track the average CPU usage over a five-minute window and set an alert if it exceeds 90%. While this approach is effective for catching known issues, it lacks the context needed to understand why a problem is occurring.
+
+Observability: High-Fidelity, High-Granularity and Context-Rich Metrics
```
Comment (hugovalente-pm) [inline blog/2023-10-24-monitoring-vs-observability/index.md:53]:
```suggestion
Observability: High-Fidelity, High-Granularity and Context-Rich Metrics

```

missing paragraph

**Pull request merged**